packages/terra-grid/src/GridColumn.jsx

```diff
@@ -27,9 +27,13 @@ const propTypes = {
   /**
    * The number of columns across all breakpoints. From 1-12(inclusively).
    */
-  col: columnRange,
+  tiny: columnRange,
```

Comment:
Should tiny be required since it'll be the default?

Reply:
This should stay as an optional prop.